backend/FwLite/LcmCrdt/CrdtMiniLcmApi.cs

```diff
@@ -182,8 +215,12 @@ private async Task LoadSenses(Entry[] entries)
                 .Where(e => senses.Select(s => s.Id).Contains(e.SenseId)).ToArrayAsyncEF())
             .ToLookup(e => e.SenseId)
             .ToDictionary(g => g.Key, g => g.ToArray());
+
+        //could optimize this by doing a single query, but this is easier to read
+        entry.Components = [..await ComplexFormComponents.Where(c => c.ComplexFormEntryId == id).ToListAsyncEF()];
+        entry.ComplexForms = [..await ComplexFormComponents.Where(c => c.ComponentEntryId == id).ToListAsyncEF()];
```

Comment:
This seems like a pretty small legibility sacrifice to make for a whole db query 🤷
```suggestion
        var complexFormComponents = await ComplexFormComponents.Where(c => c.ComplexFormEntryId == id || c.ComponentEntryId == id).ToListAsyncEF();
        entry.Components = [..complexFormComponents.Where(c => c.ComplexFormEntryId == id)];
        entry.ComplexForms = [..complexFormComponents .Where(c => c.ComponentEntryId == id)];
```